cluster-controller/src/test/java/io/strimzi/controller/cluster/resources/TopicControllerTest.java

```diff
@@ -54,6 +56,7 @@
         expected.add(new EnvVarBuilder().withName(TopicController.KEY_NAMESPACE).withValue(tcNamespace).build());
         expected.add(new EnvVarBuilder().withName(TopicController.KEY_FULL_RECONCILIATION_INTERVAL_MS).withValue(tcReconciliationInterval).build());
         expected.add(new EnvVarBuilder().withName(TopicController.KEY_ZOOKEEPER_SESSION_TIMEOUT_MS).withValue(tcZookeeperSessionTimeout).build());
+        expected.add(new EnvVarBuilder().withName(TopicController.KEY_TOPIC_METADATA_MAX_ATTEMPTS).withValue(String.valueOf(tcTopicMetadataMaxAttempts)).build());
```

Comment:
I think I've said this before, but we should factor out a utility function for creating `EnvVar`s like this. We're slowing accruing a lot of duplicated code.

Reply:
Yes. You are highlighting it in the tests classes where an utility method could be put inside `ResourceUtils` class. Btw we have the same problem in the resource classes as well (i.e. `TopicController`, `KafkaCluster` and so on) so the utility method could live in the base `AbstractCluster` class. Wdyt ?

Reply:
Yeah, I think a `static` method in `AbstractCluster` is fine for now.

Reply:
I did it and used the method just in the `TopicController` (and related test). Of course I want to use this method in the other resource classes as well but in a separate PR.